samples/TestApp.PCL/TestApp.PCL.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard1.1</TargetFramework>
+	<DebugType>Full</DebugType>
```

Comment:
I've updated the resulting error message and there will be absolutely no ambiguity about why it doesn't work or how to fix it.

We haven't announced support for the format either, so I don't think it's necessarily great to announce all the new stuff we happen to be temporarily incompatible with. Especially with something that had been such a moving target as netcore/netstandard.

I.e. I don't think it would be very useful to explicit call out all the things you had to manually tweak to get the whole netstandard project working in the first place.